tsnet/tsnet_test.go

```diff
@@ -997,6 +1087,18 @@ func TestUserMetrics(t *testing.T) {
 		t.Errorf("metrics1, tailscaled_primary_routes: got %v, want %v", got, want)
 	}
 
+	// Verify that the amount of data recorded in bytes is higher than the
```

Comment:
In all 4 of these checks the comment and the error message do not match the condition - e.g. this one says "higher" but you are really checking for "not lower" (i.e. "higher or equal"). I'd suggest making these consistent.